### PR TITLE
[Android] Fixing PayPal without Drop-in

### PR DESCRIPTION
**Pull request by muccy**
- I fixed Android example project: now it compiles.
- I changed the README in order to instruct about how to write a Manifest.xml that allows to request a PayPal payment without using drop-in
- I fixed a bug during cancellation caused from strange SDK behaviour

Despite what's written in commit messages, I didn't update SDK version.

I'm still under the strong impression that paying with PayPal drop-in doesn't work, but I hadn't the time to fix also that problem (sorry)

### CODE REVIEW

**Comment by muccy** [comment]
I've just completed the PayPal request parameters on Android, too

**Comment by wwwdata** [comment]
Hm, this PR is open for a long time now :D Is this plugin still maintained?

Could you summarize again quickly why you made those changes? When I test out the Custom Activity with browser-switch it all works fine without your changes. But thanks a lot for the documentation of how to properly configure the Custom Activity.

It also still works after upgrading to the latest drop-in sdk without your changes #145

**Comment by muccy** [comment]
Hi,
I made these changes because I needed PayPay payment but not using drop-in. At that time, Android app displayed a blank activity and then was not able to process the payment.
I don't know if upgrading fixes all the problems now. Maybe I'll try, maybe not because the app that uses these changes is already in production for thousands of people. Thanks a lot for the info

**Comment by wwwdata** [comment]
Ok, I am also using your PR now with the latest drop-in version from Braintree. I had some issues/crashes before on some devices and your changes fixed them for me.

Let's hope that some day this will be merged in.
